webui/react/src/components/Searches/columns.ts

```diff
@@ -143,97 +150,106 @@ export const getColumnDefs = ({
   checkpointSize: {
     id: 'checkpointSize',
     isNumerical: true,
-    renderer: (record: ExperimentWithTrial) => ({
-      allowOverlay: false,
-      copyData: record.experiment.checkpointSize
-        ? humanReadableBytes(record.experiment.checkpointSize)
-        : '',
-      data: { kind: TEXT_CELL },
-      kind: GridCellKind.Custom,
-    }),
+    renderer: (record: ExperimentWithTrial) =>
+      record.experiment.checkpointSize !== undefined
+        ? {
+            allowOverlay: false,
+            copyData: humanReadableBytes(record.experiment.checkpointSize),
+            data: { kind: TEXT_CELL },
+            kind: GridCellKind.Custom,
+          }
+        : EMPTY_CELL,
     title: 'Checkpoint Size',
     tooltip: () => undefined,
     width: columnWidths.checkpointSize,
   },
   description: {
     id: 'description',
-    renderer: (record: ExperimentWithTrial) => ({
-      allowOverlay: false,
-      copyData: String(record.experiment.description),
-      data: { kind: TEXT_CELL },
-      kind: GridCellKind.Custom,
-    }),
+    renderer: (record: ExperimentWithTrial) =>
+      record.experiment.description
```

Comment:
```suggestion
      record.experiment.description !== undefined
```
i think we wanna show empty string as it is instead of `-`, right? because empty string doesnt necessarily mean no data. or is description a special case?

Reply:
I don't think that the description ever actually gets set to `undefined`. So yeah, a special case.